src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -7620,6 +7620,10 @@ ExpressionSyntax node4
             Assert.Equal(MethodKind.BuiltinOperator, symbol1.MethodKind);
             Assert.True(symbol1.IsImplicitlyDeclared);
 
+            var synthesizedMethod = compilation.CreateBuiltinOperator(
+                symbol1.Name, symbol1.ReturnType, symbol1.Parameters[0].Type, symbol1.IsCheckedBuiltin);
+            Assert.Equal(synthesizedMethod, symbol1);
```

Comment:
hooks to validate existing testcases where we create builtin-operators can properly be created with this new api.